File: 125-valid-palindrome/125-valid-palindrome.cpp
class Solution {
public:
    bool isPalindrome(string s) {
         transform(s.begin(), s.end(), s.begin(), ::toupper);
        string s3;
        for(auto ch:s){
             if(isalnum(ch))
                s3+=ch;
        }
        string s2=s3;
            reverse(s3.begin(), s3.end());
        return s3==s2;
        
    }
};

File: 125-valid-palindrome/125-valid-palindrome.js
/**
 * @param {string} s
 * @return {boolean}
 */
var isPalindrome = function(s) {
    s=s.toLowerCase().replace(/[^A-Za-z0-9]/g, "");
    
    let t=s.split("").reverse().join("");

    return s==t;
};